src/test/java/org/gridsuite/study/server/VoltageInitTest.java

```diff
@@ -111,6 +111,8 @@ public class VoltageInitTest {
 
     private static final String VOLTAGE_INIT_RESULT_JSON = "{\"version\":\"1.0\"}";
 
+    private static final String VOLTAGE_INIT_PREVIEW_MODIFICATION_LIST = "[{\"type\": \"VOLTAGE_INIT_MODIFICATION\",\"uuid\": \"254a3b85-14ad-4436-bf62-3e60af831dd1\",\"date\": \"2023-09-18T10:58:32.582239Z\",\"stashed\": false,\"generators\": [],\"transformers\": [],\"staticVarCompensators\": [],\"vscConverterStations\": []}]";
+
```

Comment:
Add shuntCompensators in the json, as the US to retrieve the shunt compensators modifications has been merged.